src/errors.rs

```diff
@@ -11,7 +11,23 @@ use thiserror::Error;
 use crate::{renderer::render_error, MiniserveConfig};
 
 #[derive(Debug, Error)]
-pub enum ContextualError {
+pub enum StartError {
```

Comment:
Let's go with this instead.
```suggestion
pub enum StartupError {
```